CONTRIBUTING.md

```diff
@@ -1,61 +1,66 @@
 # Contributing to MUI
 
-If you're reading this, you're awesome! Thank you for helping us make this project great and being a part of the MUI community. Here are a few guidelines that will help you along the way.
+If you're reading this, you're awesome!
+Thank you for being a part of the MUI community and helping us make this project great.
```

Comment:
```suggestion
Thank you for being a part of the MUI community and helping us make these projects great.
```

Maybe plural here to refer to all Core products instead of Material UI?